infrastructure/ecs-main/ecs_service.yml

```diff
@@ -112,6 +112,8 @@ Resources:
               ValueFrom: arn:aws:ssm:${aws:region}:${aws:accountId}:parameter/env/${self:provider.stage}/docs/worker_pool/atlas/host
             - Name: DB_NAME
               ValueFrom: arn:aws:ssm:${aws:region}:${aws:accountId}:parameter/env/${self:provider.stage}/docs/worker_pool/atlas/dbname
+            - Name: SNOOTY_DB_NAME
+              ValueFrom: arn:aws:ssm:${aws:region}:${aws:accountId}:parameter/env/${self:provider.stage}/docs/worker_pool/atlas/collections/snooty
```

Comment:
this has been added to the parameter store via AWS console. can verify them at https://us-east-2.console.aws.amazon.com/systems-manager/parameters/?region=us-east-2&tab=Table#list_parameter_filters=Name:Contains:%2Fdocs%2Fworker_pool%2Fatlas%2Fcollections%2Fsnooty